### DISCUSSION TITLE
global variables on receiver device hang sometimes

### DISCUSSION THREAD

**Discussion by HofeBY** [arendst/Tasmota#21910] [Q&A]
I have a question about the use of global variables on receiver device.

I use the global variable to transmit the outside temperature or the
current power of the electricity meter.

This works quite easily with a couple of receiver devices.
But sometimes this connection goes down and the last value remains on the receiver devices.
Other receiver devices still get this value at the same time.
The global variable transmitter module still has changing values, which I can also see in the mqtty streamer.

By rebooting the stopped receiver modules, it works again until the next hang.
Would it be possible to restart the connection after about 5 minutes with the same global variable value in order to get the current values ​​again without restarting the receiver device?
If this connection is not established, a null value or invalid character would not be a bad idea.

**Comment by gemu2015** [comment]
this is a known problem and i just saw that the global var receiver reset cmd is not yet in the docs
`gvr` resets the global receiver
section >G is called when a global var is received.  you may use this to reset a timer and when this timer expires call `gvr` or call `gvr` at regular intervals

**Reply by HofeBY** [reply]
realy fast response. thx
but gvr command say´s

17:05:33.405 CMD: gvr
17:05:33.412 MQT: stat/22ESP32/RESULT = {"Command":"Unknown"}

do I have to set something to include it to my custom build TM 14.1.0.2

**Reply by gemu2015** [reply]
not in console, but in script

**Reply by HofeBY** [reply]
OK, I try & after some day´s I can say it help´s.
Implemented to reconnect every 5min. waiting for new stop.

. >G

.>S
if upsecs%300==0
{
gvr
}
